src/OMSimulatorLib/Model.cpp

```diff
@@ -587,6 +595,25 @@ oms_status_enu_t oms::Model::emit(double time, bool force)
   if (!force && time < lastEmit + loggingInterval)
     return oms_status_ok;
 
+
+  SignalValue_t wallTime;
+  wallTime.realValue = clock.getElapsedWallTime();
+  resultFile->updateSignal(clock_id, wallTime);
+  if(getTopLevelSystem()->getSolver() == oms_solver_wc_mav)
+  {
+    SignalValue_t stepS;
+    stepS.realValue = stepSize;
+    resultFile->updateSignal(h_id,stepS);
+    SignalValue_t rollB;
+    rollB.intValue = rollBackIt;
+    resultFile->updateSignal(roll_iter_id,rollB);
+    SignalValue_t maxErr;
+    maxErr.realValue = maxError;
+    resultFile->updateSignal(max_error_id,maxErr);
+    SignalValue_t errNorm;
+    errNorm.realValue = normError;
+    resultFile->updateSignal(error_id,errNorm);
```

Comment:
This additional information shouldn't be available by default. Please add a flag to enable it.

Reply:
… and they need to be moved to oms::SystemWC